src/main/java/task/Deadline.java

```diff
@@ -0,0 +1,56 @@
+package task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Represents a deadline task.
+ */
+public class Deadline extends Task {
+    private static final DateTimeFormatter INPUT_FORMATTER = DateTimeFormatter.ofPattern("yyyy-MM-dd HHmm");
+    private static final DateTimeFormatter OUTPUT_FORMATTER = DateTimeFormatter.ofPattern("MMM dd yyyy, h:mm a");
```

Comment:
INPUT_FORMATTER and OUTPUT_FORMATTER are apt names for this constants. Well done!